addons/hr_timesheet/views/project_views.xml

```diff
@@ -340,6 +340,17 @@
             </field>
         </record>
 
+        <record id="view_task_search_form_hr_extended" model="ir.ui.view">
+            <field name="name">project.task.view.search.inherit.hr.timesheet</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_task_search_form_extended"/>
+            <field name="arch" type="xml">
+                <xpath expr="//filter[@name='my_favorite_projects']" position='after'>
+                    <filter string="My Team's Projects" name="my_teams_projects" domain="[('project_id.tasks.user_ids.employee_parent_id.user_id', '=', uid)]"/>
```

Comment:
Why don't you use the user_ids of the current task? 🤔 
This domain seems a bit overkill, I think.
```suggestion
                    <filter string="My Team's Projects" name="my_teams_projects" domain="[('user_ids.employee_parent_id.user_id', '=', uid)]"/>
```

Reply:
If I do this it would work like` My Team's Tasks`  filter.

Reply:
I don't agree with you. 😅
If we see all tasks of a project, you will check for a task if all the tasks (also the archived ones) in the project has a user who has the current one as manager, if it is the case then we take the task. In resume, we will see all the tasks of the project or nothing. 🤯

I will check this with AJU.

Reply:
Okay I understand, don't hesitate to ask the question when you didn't understand a point or if you are not sure to have correctly understood. ^^
In fact she want to have the tasks in which the current user is the manager of the user of the project related.
So, we would have something like that:
```suggestion
                    <filter string="My Team's Projects" name="my_teams_projects" domain="[('project_id.user_id.employee_parent_id.user_id', '=', uid)]"/>
```